WordPress/UITestsFoundation/Globals.swift

```diff
@@ -102,3 +102,9 @@ extension ScreenObject {
         return self
     }
 }
+
+public enum Apps {
+
+    public static let safari = XCUIApplication(bundleIdentifier: "com.apple.mobilesafari")
+    public static let springboard = XCUIApplication(bundleIdentifier: "com.apple.springboard")
```

Comment:
This was called `homeApp` before, but the "technical" term is springboard and I think it's valuable to be precise in this case. 

Additional info [here](https://www.theiphonewiki.com/wiki//System/Library/CoreServices/SpringBoard.app).